artifact.md

```diff
@@ -1,74 +0,0 @@
-# OCI Artifact Manifest Specification
-
-The goal of the Artifact Manifest Specification is to define content addressable artifacts in order to store them along side container images in a registry.
-Like [OCI Images](manifest.md), OCI Artifacts may be referenced by the hash of their manifest.
-Unlike OCI Images, OCI Artifacts are not meant to be used by any container runtime.
-
-Examples of artifacts that may be stored along with container images are Software Bill of Materials (SBOM), Digital Signatures, Provenance data, Supply Chain Attestations, scan results, and Helm charts.
```

Comment:
Would like to see language strengthening Image.md to ensure that registries MUST permit non-layer blobs to be uploaded. It's implied by requiring `application/vnd.oci.artifact.manifest.v1+json`, but under this proposal a registry could prohibit all non-layer blobs, all non-image manifests.

They would be technically conformant with OCI 1.1 as written, but referrers API would be useless or require clients to hack around limitations by wrapping artifacts in layer formatted tarballs.